src/netScore.ts

```diff
@@ -61,14 +61,7 @@ export class NetScore extends Metrics {
 
         // check if netscore is between 0 and 1
         if (this.netScore < 0 || this.netScore > 1) {
-            console.log(`NetScore out of bounds: ${this.netScore}`);
-
-            // print the metrics
-            console.log(`BusFactor: ${this.busFactor.busFactor}`);
```

Comment:
why are we deleting this? This should be Debug? There was significant issues in calculating NetScore, so knowing what its working with (the metrics) is important for debug.